epub33/overview/index.html

```diff
@@ -110,7 +152,7 @@ <h2>Package Document</h2>
 					defines a reading order for linear consumption, and associates metadata and navigation
 					information.</p>
 
-				<p>The Package Document represents a significant improvement on a typical Web site. A Web site, for
+				<p>The Package Document represents a significant extension to a typical Web site. A Web site, for
```

Comment:
I'm not sure "extension" is the right word here. The original wording is controversial, but we're neither improving nor extending web sites. We're sort of building a kind of better site map. I'd have to think about this some more, but my first thought is maybe it should say something along the lines of: "The Package Document represents a unique approach to the traditional structuring of a typical Web site to facilitate the authoring of digital publications."

Reply:
I am not sure the 'unique approach' is the right wording there, it somehow smells some sort of an exclusivity. But I agree that the word extension isn't good either. What about

> The Package Document defines a layer on top of the traditional structuring of a typical Web site to facilitate the authoring of digital publications

Reply:
works for me.